README.md

```diff
@@ -1241,23 +1244,28 @@ and the Dfinity project. [\[18\]][18]
 The Lightning Network is a proposed message relay network operating at a layer
 above the Bitcoin blockchain, enabling many orders of magnitude improvement in
 transaction throughput by moving the majority of transactions outside of the
-consensus ledger into so-called "payment channels". This is made possible (with
-great difficulty) by the Bitcoin scripting language, which enables parties to
+consensus ledger into so-called "payment channels".
+<!---
+lightning network exists beyond bitcoin
+--->
+Which enables parties to
 enter into stateful contracts where the state can be updated by sharing digital
 signatures, and contracts can be closed by finally publishing evidence onto the
 blockchain, a mechanism first popularized by cross-chain atomic swaps.  By
 openning payment channels with many parties, participants in the Lightning
-Network can become focal points for routing the payments of others, leading to a
-fully connected payment channel network, at the cost of capital being tied up on
-payment channels.
-
-While the Lightning Network can also easily extend across multiple independent
-blockchains to allow for the transfer of _value_ via an exchange market, it
-cannot be used to assymetrically transfer _tokens_ from one blockchain to
-another.  The main benefit of the GnuClear network described here is to enable
-such direct token transfers.  That said, we expect payment channels and the
-Lightning Network to become widely adopted along with our token transfer
-mechanism, for cost-saving and privacy reasons.
+Network can become focal points for routing the payments, and contracts of others, leading to a
+fully connected channel network, at the cost of capital being tied up on
+payment channels. This cost can be completely removed by making it into
```

Comment:
How is putting it in a channel going to help, if you can spend it?  Why would anyone want to get in a channel with you when you might get the money slashed?

>  When value is inside of a less-liquid location, the owner is losing value by the interest rate.

Eh, except when the less-liquid thing is earning you revenue.  We're arguing about definitions here, I don't call that "losing value by interest rate".  It's about opportunity cost, and the price of gnuts will reflect that.  There's a huge difference between parking US dollars as collateral, vs holding gnuts.  The former clearly incurs costs due to the inflation of the dollar and opportunity costs, the latter doesn't.  There's nothing better for you to do with your gnuts than stake them.  You can sell them and do something else with your money, but if you're a good validator and the market is efficient, you won't be able to do better than keep your gnuts staked.

I'm pretty sure the confusion is due to you assuming that gnuts are like currency.  It is not.

> If we can't spend from the reserve pool, then the money is already deleted.

Yes, there should be a mechanism to get the money out of the reserve pool.

> Too many bonds for stopping double-singing, but too few to stop retirement attacks.

I don't understand why you're making this distinction.  The point of preventing double-signing is to prevent retirement attacks.

> If an amount of value equal to 1/2 (the value of the reserve pool + the total value of the gnuts) should ever flow through a single block, it will be profitable for a coalition to double-sign.

Well, yeah.  Don't do that.  Anyone can figure out how to transfer funds over time to prevent such a thing.  The amount at stake determines how much volume is safe to transfer, and thus more stake == more volume, and more fees.